Automattic/wc-calypso-bridge
# Setup checklist: Header/footer, hide completed, "I'm done"

Closes #64.

This PR adds a header to the setup page, based on the mock up. It also adds an "I'm done" button that will dismiss the entire page. The "hide/show" completed task functionality present in the Calypso component also works now.

I've added a bit of code to turn Calypsoify on if the check list is visited directly. The list will be linked to from an onboarding email, and if Calypsoify somehow  gets turned off, it would be good to turn it back on.

<img width="826" alt="screen shot 2018-11-01 at 2 20 08 pm" src="https://user-images.githubusercontent.com/689165/47872915-379c3200-dde6-11e8-99d2-4f7f39e0337b.png">

<img width="822" alt="screen shot 2018-11-01 at 2 20 15 pm" src="https://user-images.githubusercontent.com/689165/47872925-3ec34000-dde6-11e8-91ba-7e8dc704369f.png">

<img width="827" alt="screen shot 2018-11-01 at 2 20 20 pm" src="https://user-images.githubusercontent.com/689165/47872935-44b92100-dde6-11e8-9023-dfde156ea47b.png">

To Test:
* Disable Calypsoify.
* Visit `wp-admin/admin.php?page=wc-setup-checklist` directly, verify Calypsoify is turned on and you can see the checklist.
* Note the new header.
* Test the hide/show functionality.
* Test the "I'm done" button in the footer. You should be redirected to the orders page (later we can make this wc-admin).
